node_modules/oae-preview-processor/lib/api.js

```diff
@@ -507,18 +507,18 @@ var _generatePreviews = function(ctx, callback) {
         if (err) {
             return callback(err);
         } else if (processor) {
-            processor.generatePreviews(ctx, ctx.content, function(err, ignored) {
+            processor.generatePreviews(ctx, ctx.content, function(err, status) {
                 if (err) {
-                    ctx.setStatus('error', callback);
-                } else if (ignored) {
-                    ctx.setStatus('ignored', callback);
+                    ctx.setStatus(ContentConstants.previews.ERROR, callback);
+                } else if (status && _.contains(ContentConstants.previews, status)) {
+                    ctx.setStatus(status, callback);
```

Comment:
What does this case concretely represent? Is there a way to check for it more directly, or if not add some code comments to explain?